Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -35,44 +103,44 @@ def __init__(self, params: dict):
 
     # region Reputation calls
 
-    def ip(self, ip: str) -> dict:
+    def ip(self, ip: str, relationships: str = '') -> dict:
         """
         See Also:
             https://developers.virustotal.com/v3.0/reference#ip-info
         """
         return self._http_request(
             'GET',
-            f'ip_addresses/{ip}'
+            f'ip_addresses/{ip}?relationships={relationships}'
```

Comment:
relevant to all, if no `relationships` configured, will this call fail? let's add UT for that.

Reply:
Are you asking regarding the api call? regarding its failure?
If so, I've checked it does not fail, if no relationships are set then the api does not return them.